addons/l10n_ec/data/res_country_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <record id="base.ec" model="res.country">
+            <field name="zip_required">False</field>
```

Comment:
@stevTresCloud we already have this implemented, zip code is not required in eCommerce, so we don't need this change in this PR.
**Here is where you can check on this:**
1. [PR that introduced eCommerce made zip not required](https://github.com/odoo/odoo/pull/142730/files#diff-b74c3d7f410ba41cdaaabae451d8fefb79fd964ea9cf476b93e96696c8215206)
2. [File where you can see this in master](https://github.com/odoo/odoo/blob/master/odoo/addons/base/data/res_country_data.xml#L415)
3. Here is an screenshot of how it looks like when you installed `l10n_ec_website_sale`
![image](https://github.com/user-attachments/assets/17c122e6-13c5-4da1-a15b-5a7ead26e898)

cc' @jco-odoo

Reply:
All the rest looks good, if you want to take a look at it @jco-odoo